languages/tolk/basic-syntax.mdx

```diff
@@ -0,0 +1,333 @@
+---
+title: "Tolk basic syntax"
+sidebarTitle: "Basic syntax"
+---
+
+import { TolkHighlight } from "/snippets/tolk-highlight.jsx";
+
+<TolkHighlight />
+
+Syntax of Tolk is similar to TypeScript, Rust, and Kotlin. It is designed to be straightforward to read and write.
+
+Below is a list of basic syntax elements with examples. Most sections end with a link to a detailed topic.
+
+## Imports
+
+Imports exist at the top of the file:
+
+```tolk
+import "another-file"
+
+// symbols from `another-file.tolk` are visible
+```
+
+In typical workflows, an IDE inserts imports automatically (for example, when selecting an element from auto‑completion).
+
+Note: the entire file is imported, all its symbols are accessible. There are no "modules" or "exports", all symbols must have unique (verbose) names project-wise.
```

Comment:
Should we use `:::note :::` here for better visibility?

Reply:
okay